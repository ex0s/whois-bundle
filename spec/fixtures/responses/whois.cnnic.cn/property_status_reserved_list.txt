Sorry, The domain you requested is in the reserved list.
