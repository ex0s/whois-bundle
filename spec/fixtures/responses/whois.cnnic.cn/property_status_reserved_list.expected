#status
  %s == []

#available?
  %s == false

#registered?
  %s == false


#reserved?
  %s == true
